text/chapter5.md

```diff
@@ -463,11 +464,11 @@ origin = Point { x, y }
 
 This can be useful for improving readability of code in some circumstances.
 
- ## Exercises
+## Exercises
 
- 1. (Easy) Construct a value of type `Shape` which represents a circle centered at the origin with radius `10.0`.
- 1. (Medium) Write a function from `Shape`s to `Shape`s, which scales its argument by a factor of `2.0`, and centers it at the origin.
- 1. (Medium) Write a function which extracts the text from a `Shape`. It should return `Maybe String`, and use the `Nothing` constructor if the input is not constructed using `Text`.
+1. (Easy) Write a function `circleAtOrigin` which constructs a `Circle` (of type `Shape`) centered at the origin with radius `10.0`.
+1. (Medium) Write a function `doubleScaleAndCenter` which scales the size of a `Shape` by a factor of `2.0` and centers it at the origin.
```

Comment:
Ideally the text of each exercise mentions the name of the function to write as soon as possible.